src/MiddleWare/AdminStyle.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace PhpSchool\Website\Middleware;
+
+use PhpSchool\Website\PhpRenderer;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+
+/**
+ * @author Aydin Hassan <aydin@hotmail.co.uk>
+ */
+class AdminStyle
+{
+    /**
+     * @var PhpRenderer
+     */
+    private $renderer;
+
+    public function __construct(PhpRenderer $renderer)
+    {
+        $this->renderer = $renderer;
+    }
+
+    /**
+     * @param ServerRequestInterface $request
+     * @param ResponseInterface $response
+     * @param callable $next
+     * @return ResponseInterface
+     */
+    public function __invoke(ServerRequestInterface $request, ResponseInterface $response, callable $next)
+    {
+        $this->renderer
+            ->removeCss('main-css')
```

Comment:
scrap that actually cause I feel like the other methods would then need to do the same and they have multiple params which would just get messy 😂